src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

you're implicitly using `any` transition - that slows down the page load times
better use shorthand
```suggestion
    transition: transform 0.3s, color 0.3s;
```
same for the header transition below